.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false
+language: python
+env:
+  - AWS_REGION=localhost
+  - AWS_ACCESS_KEY_ID=AAAAAAAAAAAAAAAAAAAA
```

Comment:
I'm not even sure those are actually necessary now that I look at it. I think the failure I was hitting was due to running the tests too soon after dynamo was started - it hadn't had a chance to start listening yet.